cfpb/cfgov-refresh
# Fix two Mortgage Performance bugs: dynamic map title, invalid date notification

This PR fixes two bugs for Mortgage Performance Trends.

1) The dynamic map title was not properly resetting in the state view when a user switches from looking at a specific state to the overall state view, and
2) The notification warning for invalid dates didn't have the correct spacing from the drop downs that triggered it.

## Additions

- Add 30px margin on top of the invalid date notification.

## Changes

- Make the reset option available to the state view, as well as to the county and metro view.

## Testing

1. Pull down this branch and `./setup.sh`
2. Set up the [mortgage performance API](https://github.com/cfpb/cfgov-refresh/pull/3189).
3. Create a [mortgage chart block, mortgage map block, and page](https://github.com/cfpb/cfgov-refresh/pull/3260).
4. **For the invalid date notification:** Try to show the map using data for December 2017. There should be 30px space between the notification that appears and the date drop downs.
5. **For the dynamic map title:** Try to see the state view for New Jersey, then reset by selecting the "Please select a state" option. The dynamic title should say 'state view'.

## Checklist
